go/tools/filter_tags/filter_tags.go

```diff
@@ -29,6 +30,10 @@ var (
 	cgo   = flag.Bool("cgo", false, "Sets whether cgo-using files are allowed to pass the filter.")
 	quiet = flag.Bool("quiet", false, "Don't print filenames. Return code will be 0 if any files pass the filter.")
 	tags  = flag.String("tags", "", "Only pass through files that match these tags.")
+	run   = flag.Bool("run", false, "Run mode, filter a command line and then run it.")
+
+	absMarker    = "≡"
+	filterMarker = ""
```

Comment:
As we discussed, moving to a model that uses a longer string rather than a magic character, and a different "escaping" prefix, but keeping the rest of the model the same